### PR TITLE
usage of sparsemax in enc-dec attention task

### PR DESCRIPTION
**Pull request by alrojo**
An ipython notebook for usage test of softmax vs. sparsemax.

In the notebook, search for `attention_fn`, replace the current argument with the `sparse_max` function instead of the `tf.nn.softmax` function.

You might want to start with having the `sparse_max` outputting a dense tensor to test it works as intended.
Having the output as a [SparseTensor](https://www.tensorflow.org/versions/r0.11/api_docs/python/sparse_ops.html#sparse-tensors) might cause some issues.

### CODE REVIEW

**Comment by AndreasMadsen** [comment]
rebased in #15
